.pre-commit-config.yaml

```diff
@@ -1,19 +1,31 @@
 repos:
-- repo: https://github.com/dnephin/pre-commit-golang
-  rev: v0.5.1
-  hooks:
-    - id: go-fmt
-      exclude: ^vendor
-    - id: go-vet
-    - id: go-mod-tidy
-
-- repo: https://github.com/golangci/golangci-lint
-  rev: v1.50.1
-  hooks:
-    - id: golangci-lint
-
 - repo: local
   hooks:
+    - id: golangci-lint
```

Comment:
Need to add tidy and golangci-lint in Makefile[1] 

[1] https://github.com/openstack-k8s-operators/keystone-operator/pull/202

Reply:
Good catch thanks, adding in the next push!